.github/workflows/maven.yaml

```diff
@@ -1,54 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#       http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-
-name: Java CI
-
-on: [ push, pull_request ]
-
-jobs:
-  build:
-
-    strategy:
-      matrix:
-        os: [ ubuntu-latest, windows-latest, macOS-latest ]
-        java: [ 8, 11, 17, 21 ]
-      fail-fast: false
-
-    runs-on: ${{ matrix.os }}
-
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-
-# https://github.com/actions/setup-java/blob/main/docs/advanced-usage.md#modifying-maven-toolchains
-      - name: Install Toolchain JDK
-        if: ${{ matrix.java == '8' }}
-        uses: actions/setup-java@v4
-        with:
-          distribution: 'zulu'
-          java-version: 11
-          cache: 'maven'
```

Comment:
This one can be needed by project

Reply:
Maybe we should run on JDK 11+

Reply:
reverted the workflow files (restored maven.yaml, removed maven-verify.yaml)

Reply:
> Maybe we should run on JDK 11+

Let's discuss this in another PR/issue. I only replaced the files, because I thought this has an old build file and not the current overall one.